t/selenium/CAA.json5

```diff
@@ -152,6 +158,17 @@
         release: 'https://musicbrainz.org/release/dd245091-b21e-48a3-b59a-f9b8ed8a0469',
       },
     },
+    // Verify that the seeded edit note was used.
```

Comment:
The `/edit/<ID>/data` endpoint doesn't seem to include the original edit note, so I think I need to load the page in order to confirm that the seeded value was used. Let me know if I'm missing something, though.